pkg/wal/processor/transformer/wal_postgres_transformer_parser.go

```diff
@@ -33,6 +33,7 @@ func NewPostgresTransformerParser(ctx context.Context, pgURL string, builder tra
 
 func (v *PostgresTransformerParser) ParseAndValidate(rules []TableRules) (map[string]ColumnTransformers, error) {
 	transformerMap := map[string]ColumnTransformers{}
+	pgtypeMap := pgtype.NewMap()
```

Comment:
This should probably be a field of the `PostgresTransformerParser`, since it only needs to be initialised once. In our case, the `ParseAndValidate` is only called once, but that might not always be the case.

Worth noting as well that the custom types will not be covered by this, for those the oid will still be in the error message.